Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -303,7 +327,14 @@ namespace AZ::SceneAPI::Behaviors
 
                 // get node matrix data to set the entity's local transform
                 const auto nodeTransform = azrtti_cast<const DataTypes::ITransform*>(graph.GetNodeContent(thisTransformIndex));
-                entityTransform->SetLocalTM(AZ::Transform::CreateFromMatrix3x4(nodeTransform->GetMatrix()));
+                if (nodeTransform)
+                {
+                    entityTransform->SetLocalTM(AZ::Transform::CreateFromMatrix3x4(nodeTransform->GetMatrix()));
+                }
+                else
+                {
+                    entityTransform->SetLocalTM(AZ::Transform::CreateUniformScale(1.0f));
```

Comment:
There's probably an identity matrix function/const you could use here to make it more clear, but it's not a big deal